### PR TITLE
Merge dev changes

### PR DESCRIPTION
**Pull request by rudolphpienaar**
This completes the dev->master merge of pf-services.

@danmcp

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ pfurl - v1.3.7
 Overview
 ********
 
-This repository provides a curl-like client tool that is also able to perform simple file/directory functions (such as zip/unzip) suitable for remote http transmission:
+This the bleeding edge repository that provides a curl-like client tool that is also able to perform simple file/directory functions (such as zip/unzip) suitable for remote http transmission:
```
Comment (danmcp) [inline README.rst]: Grammar is off here.  Also I am guessing this line isn't valid now that it's moving back to master.

Comment (rudolphpienaar) [reply README.rst]: Yep! Many thanks.

Comment (rudolphpienaar) [reply README.rst]: Other than that... ok to merge as far as you can tell?

**Comment by danmcp** [comment]
@rudolphpienaar LGTM

**Pull request merged**